spec/visual/samples/lustre

```diff
@@ -0,0 +1,244 @@
+
+-------------------------------------------------------------------
+(* Compute the roof motion speed *)
+  
+const period = 0.1;  -- in seconds
+const wheel_girth = 1.4;  -- meter
+const size = 50; -- size of the sliding windows used to smooth values
+const max_roof_speed = 10.0;
+const speed_max = 110.0;                                           --
+const speed_hist= 5.0; 

```

Comment:
Could you replace some of these floats with binary, octal and hexadecimal numbers to ensure they're lexing correctly?

Reply:
Octal and hexadecimal numbers are (currently) not supported by the Lustre compiler (maybe they should).

Reply:
Ok, I will remove them from the syntax